grpc/execution/server.go

```diff
@@ -159,10 +114,18 @@ func (s *ExecutionServiceServerV1) GetGenesisInfo(ctx context.Context, req *astr
 	rollupHash := sha256.Sum256([]byte(s.bc.Config().AstriaRollupName))
 	rollupId := primitivev1.RollupId{Inner: rollupHash[:]}
 
+	fork := s.bc.Config().GetAstriaForks().GetForkAtHeight(s.bc.CurrentSafeBlock().Number.Uint64() + 1)
+	nextFork := s.bc.Config().GetAstriaForks().GetNextForkAtHeight(s.bc.CurrentSafeBlock().Number.Uint64() + 1)
+
 	res := &astriaPb.GenesisInfo{
-		RollupId:                    &rollupId,
-		SequencerGenesisBlockHeight: s.bc.Config().AstriaSequencerInitialHeight,
-		CelestiaBlockVariance:       s.bc.Config().AstriaCelestiaHeightVariance,
+		RollupId:                  &rollupId,
+		SequencerChainId:          fork.Sequencer.ChainID,
+		SequencerStartBlockHeight: fork.Sequencer.StartHeight,
+		SequencerStopBlockHeight:  fork.Sequencer.StopHeight,
```

Comment:
what is stop height for?

Reply:
conductor uses stop height to determine when it needs to stop sending soft commits, wait for firm to reach the same level, and do an internal restart to re-fetch genesis info from geth which should have the next fork config